src/components/Global/RadioButton/RadioButton.module.scss

```diff
@@ -0,0 +1,57 @@
+@import 'src/styles/abstracts/_variables';
+@import 'src/styles/abstracts/_mixins/text';
+
+.gone {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  height: 0;
+  width: 0;
+}
+.normal {
+  @include font-regular;
+  display: inline-block;
+  padding: 0.625rem 1.25rem;
+  margin: 0.625rem 1.25rem;
+  border: 0.125rem solid;
+  border-radius: 0.625rem;
+  text-align: center;
+  cursor: pointer;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+.easy {
+  @extend .normal;
+  border-color: $green;
+  color: $green;
+}
+.medium {
+  @extend .normal;
+  border-color: $secondary-100;
+  color: $secondary-100;
+}
+.hard {
+  @extend .normal;
+  border-color: $red;
+  color: $red;
+}
+.easyChecked {
+  @extend .easy;
```

Comment:
`@extend` here isn't necessary, you could just add or remove the `easyChecked` class and leave the `easy` class on the component. The current way you have means the `easy` styles are duplicated when an element has both `easy` and `easyChecked`

This applies to `mediumChecked` and `hardChecked`